courses/machine_learning/deepdive2/structured/solutions/1_explore_babyweight.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<p>I'd move the comment in a markdown cell above instead of having it in the code, and I'd rephrase it as follow:</p><p> </p><p>"Let's now create a SQL to retrieve the natality dataset after the year 2000:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/665/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done</p>